topparser/pom.xml

```diff
@@ -42,7 +42,8 @@
         </dependency>
         <dependency>
             <groupId>org.geotools</groupId>
-            <artifactId>gt-epsg-hsql</artifactId>
+            <artifactId>gt-epsg-wkt</artifactId>
+            <version>${geotools.version}</version>
```

Comment:
We moeten even kijken of deze doordruppelt naar de web apps en de brmo-loader, dan moet er een exclude bij, of hier de scope aanpassen, of...

Reply:
fixed met fbd901ba357139e0bbe2cf3873cc2bc2150c8654